package.json

```diff
@@ -20,5 +30,28 @@
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-mocha": "^4.7.0",
     "eslint-plugin-react": "^6.8.0"
+  },
+  "devDependencies": {
+    "babel-eslint": "^7.2.3",
+    "condition-circle": "^1.5.0",
+    "cracks": "^3.1.2",
```

Comment:
In this case, cracks would help semrel by running the `lint` commands in the sdk and the widget repos. I don't see anywhere that that's configured.

Reply:
package.json
```
"verifyRelease": {
      "path": "cracks",
      "paths": ["tests", "package.json"],
      "silent": true
    }
```

in package.json cracks is specified as a plugin. This is following the setup from https://github.com/semantic-release/cracks under usage.

@ianwremmel 
Is this what you meant?

Reply:
oh, interesting. https://github.com/semantic-release/semantic-release/issues/65 is what i was referring to but apparently it doesn't exist yet. apparently i've been talking about the wrong package. yes, this looks like the right config for cracks but for purposes of this package, we can't actually rely on it (since we can't really test it). let's pull cracks out; semantic-release will have to rely on just the commit history.